README.md

```diff
@@ -6,11 +6,11 @@ Vagrant + VBox for plutus playground
 3. When it finished provisioning run: `vagrant ssh`
 4. Run: `tmux`
 5. Add a new pane: `C + "` (`C = Ctrl + b`) / Move up/down `C + up/down`
-6. Pane1 - Start Backend: `cd /home/vagrant/plutus/git/plutus` => `nix-shell` => `cd plutus-playground-client` => `plutus-playground-server`
+6. Pane1 - Start Backend: `cd /home/vagrant/plutus/git/plutus` => `nix-shell` => `cd plutus-playground-server` => `plutus-playground-server`
```

Comment:
actually in Lars video he starts the server from plutus-playground-client and you can start them from both